layers/+lang/latex/funcs.el

```diff
@@ -20,6 +20,14 @@
     ;; (when build-proc
     ;;   (set-process-sentinel build-proc 'latex//build-sentinel))))
 
+(defun latex/all ()
+  "Save, build and view in one command."
+  (interactive)
+  (progn
+    (let ((TeX-save-query nil))
+      (TeX-save-document (TeX-master-file)))
```

Comment:
Oh, looks like `TeX-save-document` can deal with that. Neat.

Though I'm still not sure that I want a command that saves (other) files without explicit consent.